### DISCUSSION TITLE
migration

### DISCUSSION THREAD

**Discussion by alexkobylansky** [twbs/bootstrap#31356] [General]
Hello!
How easy will it be to migrate from version 4 to version 5?

**Comment by techboyg5** [comment]
While it's not as breaking as the Bootstrap 3 to 4 transition, you still have to change some things, like card decks.

**Comment by alexkobylansky** [comment]
Ok. Bootstrap v4 uses jQuery, v5 doesn't.
If I start a project now using Bootstrap v4, how easily can I ditch jQuery and keep functionality when I start using Bootstrap v5?

**Comment by techboyg5** [comment]
> Ok. Bootstrap v4 uses jQuery, v5 doesn't.
> If I start a project now using Bootstrap v4, how easily can I ditch jQuery and keep functionality when I start using Bootstrap v5?

You can still use jQuery in Bootstrap v5.

**Comment by alexkobylansky** [comment]
Yes. But I don't want it ))
I was asked not to use Bootstrap v4 precisely because of the use of jQuery

**Comment by ffoodd** [comment]
v5 is only in its early stages, there are breaking changes incoming — so no one can really answer this for now.

FYI, there's a migration guide that's updated alongside any change, you'll have to refer to it when migrating.

**Comment by alexkobylansky** [comment]
Ok, thanks for your answer.
I will look forward to new innovations in the fifth version.
Your project has always been ambitious, and I believe that the fifth version will be better than all the previous ones.

**Comment by mc2002tii** [comment]
> 
> Ok. Bootstrap v4 uses jQuery, v5 doesn't.
> If I start a project now using Bootstrap v4, how easily can I ditch jQuery and keep functionality when I start using Bootstrap v5?

I just converted one of my own projects from v4 to v5-alpha1 and removed jQuery from the code.  Probably 98% of my time was spent ripping jQuery out of my code; converting the bootstrap calls to vanilla js was pretty straightforward.

**Reply by alexkobylansky** [reply]
Maybe take a chance and immediately launch the project on the fifth version?
